docs/actions.md

```diff
@@ -567,16 +570,16 @@ The action may be invoked as any other OpenWhisk action.
   }
   ```
 
-If you update the Docker image, run `buildAndPush.sh` to refresh the image on Docker Hub, then run `wsk action update` to make the system to fetch the new image. New invocations will start using the new image and not a warm image with the old code.
-
+  To update the Docker image, run `buildAndPush.sh` to refresh the image on Docker Hub, then run `wsk action update` to make the system to fetch the new image. New invocations will start using the new image and not a warm image with the old code.
```

Comment:
@rabbah Please provide the text you would like to see here to make clear to the user.

My impression was that the only thing users need to worry about when they have a new image for the same action was to run `wsk action update`, but it looks they have to do something else?